README.md

```diff
@@ -228,6 +228,23 @@ Customize the Docker container run process by adding properties under the `docke
 | `env` | Environment variables applied to the container. | None |
 | `envFiles` | Files of environment variables read in and applied to the container. Environment variables are specified one per line, in `<name>=<value>` format. | None |
 | `labels` | The set of labels added to the container. | `com.microsoft.created-by` = `visual-studio-code` |
+| `ports` | Ports that are going to be mapped on the host. | All ports exposed in the Dockerfile will get binded to a random port on the host machine |
+| `extraHosts` | Hosts to be added on the container `hosts` file for dns resolution. | None |
+| `volumes` | Volumes that are going to be mapped to the container. | None |
+
+# ports
+| Property | Description | Required |
+| --- | --- | --- |
```

Comment:
Any reason to use a `Required` column here for ports but a `Default` column for volumes? Could we use the same column in both tables?

Reply:
Not a good reason at all, to make it consistent I'll apply the same colums for both tables like the following:

| Property | Description | Required | Default |
| --- | --- | --- | --- |